src/Commands/SyncCommand.php

```diff
@@ -115,4 +170,20 @@ private function buildDefault(string $type, $default)
 
         return $default;
     }
+
+    private function validateUpdateFields(): void
+    {
+        if (! $this->option('update-fields')) {
```

Comment:
Se podría reemplazar por `! $this->hasOption('update-fields')` para que sea más semántico.